bundles/ranvier-areas/areas/limbo/rooms.yml

```diff
@@ -1,113 +1,116 @@
-- id: 1
-  title: Test Room 1
+- id: white
+  title: "White Room"
   description: "A featureless white room. A pitch black void in the shape of archway can be seen on the east side of the room."
-  npcs: ["limbo:1" ]
+  npcs: ["limbo:rat" ]
   items:
-    - id: "limbo:3"
+    - id: "limbo:woodenchest"
       respawnChance: 20
       replaceOnRespawn: true # when this chest respawns completely replace it so its contents get refreshed
   script: "1-test"
```

Comment:
I suggest changing the script names to match the room/item ids in here and for items. The scripts live in `ranvier-areas/areas/limbo/scripts/ there